README.md

```diff
@@ -51,8 +47,14 @@ Using Refract, we can expand this out and add some human-readable titles and des
     {
```

Comment:
To be honest, I think JSON may be the best way to express this structure as it is very detailed and contains a lot of information. I've left it in JSON, especially since the future paragraphs talk about "JSON" in comparison to this structure.

I have updated the serialisation of the meta components to follow JSON serialisation rules.